PostFreeSurfer/PostFreeSurferPipeline.sh

```diff
@@ -141,6 +161,27 @@ FreeSurferFolder="$T1wFolder"/"$FreeSurferFolder"
 AtlasTransform="$AtlasSpaceFolder"/xfms/"$AtlasTransform"
 InverseAtlasTransform="$AtlasSpaceFolder"/xfms/"$InverseAtlasTransform"
 
+
+# ------------------------------------------------------------------------------
+#  Compliance check
+# ------------------------------------------------------------------------------
+
+MPPMode=`opts_GetOpt1 "--mpp-mode" $@`
+MPPMode=`opts_DefaultOpt $MPPMode "HCPStyleData"`
+Compliance="HCPStyleData"
+ComplianceMsg=""
```

Comment:
Not sure how you've structured all the other scripts, but shouldn't the parsing of the `--mpp-mode` value, and the setting of its default value, occur earlier when that is done for all the other variables?  And shouldn't the value of `--mpp-mode` then be included as one of the `verbose_echo` statements?

Reply:
It is placed here because it depends on the definition of `T2wFolder` and I wanted to have everything related to compliance checking in one place. But you are right. Your suggestion makes sense. I'll make the change.